tests/regression/src/lib.rs

```diff
@@ -236,16 +245,16 @@ mod tests {
             format!("target/diff/{}.diff", self.test_name)
         }
 
-        fn assert_performance(&self) {
+        fn assert_performance(&self, max_diff: f64) {
```

Comment:
I wonder if separating these out to a const/static might keep the thresholds more maintainable/visible? Like essentially all of the threshold configuration is just a map from test_name -> threshold.

Reply:
Hmmm, I think I understand. Like having a global static map that you would update every time a new harness is added? 
`lazy_static! {
    pub static ref CONFIG_MAP: HashMap<&'static str, f64> = {
        let mut m = HashMap::new();
        m.insert("test_set_config", 0.01);
        m.insert("test_rsa_handshake", 0.01);
        m
    };
}`
I guess that doesn't add much complexity to the workflow of adding a harness and it assembles all thresholds in one place, so you can see how they all stack up in comparison so I am in favor of it.

Reply:
Why wouldn't the threshold in the same place as the harness? I'm not really a fan of having a global config as it's less obvious how to add new harnesses for devs